server/datastore/mysql/software_installers.go

```diff
@@ -1943,7 +1943,7 @@ SELECT
   id
 FROM
   software_titles
-WHERE (unique_identifier, source, extension_for) IN (%s)
+WHERE (unique_identifier, source) IN (%s) AND extension_for = ''
```

Comment:
This change is because we do not deploy extensions via GitOps right?

Reply:
Correct. If you look at the query params changes later on in the PR, we had `""` for the extension_for field so this just inlines it and drops the number of bound parameters by one per title.